klaviyo/lists.py

```diff
@@ -0,0 +1,214 @@
+from .api_helper import KlaviyoAPI
+
+
+class Lists(KlaviyoAPI):
+    LIST = 'list'
+    LISTS = 'lists'
+    SUBSCRIBE = 'subscribe'
+    MEMBERS = 'members'
+    ALL = 'all'
+    LIST_NAME = 'list_name'
+
+    def get_lists(self):
+        """Returns a list of Klaviyo lists
+        https://www.klaviyo.com/docs/api/v2/lists#post-lists
```

Comment:
double nit: incorrect url and a newline after summary docstring between additional info